### DISCUSSION TITLE
代码块扩展语法 - 图表支持 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#523] [General]
# 代码块扩展语法 - 图表支持 | FixIt

这篇文档将展示如何在 FixIt 中使用代码块扩展语法创建 SVG 和 Canvas 图表。

https://fixit.lruihao.cn/zh-cn/docs/content-management/diagrams/

**Comment by liangml** [comment]
推荐将mermaid cdn链接改成：https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.esm.min.mjs 10版本的链接加载太慢了

**Reply by Lruihao** [reply]
好的
